apps/roam/src/utils/fireQuery.ts

```diff
@@ -347,11 +371,31 @@ const fireQuery: FireQuery = async (_args) => {
       console.groupEnd();
     }
 
-    //@ts-ignore - todo add async q to roamjs-components
-    const queryResults = await window.roamAlphaAPI.data.backend.q(
-      query,
-      ...inputs,
-    );
+    let queryResults: unknown[][] = [];
+    if (local) {
+      // look for propNames in query. Could consider looking only in pull when that exists.
```

Comment:
Could you clarify what this is looking for?  What does `propNames` refer to and/or why are we checking for it?